packages/web-components/src/components/rux-tree-node/rux-tree-node.tsx

```diff
@@ -23,6 +23,8 @@ let id = 0
 /**
  * @slot (default) - The parent node content
  * @slot node - Renders a child node within the current node
+ * @part item - The individual tree node
```

Comment:
I personally like 'node' here more than item. I don't see an issue with overlapping the naming with the slot. Since it's called 'rux-tree-node', just seems more natural that the part you'd be styling is the 'node'. 

Again, I don't want to block this with something like that lol. @markacianfrani whaddya think

Reply:
ya I agree I think `node` is better of the two

Reply:
Happy to change it :) we'll have to change it from item to node in Brian's branch for next as well. I'll ping him about it since I don't know where he's at on that one.